sympy/functions/elementary/integers.py

```diff
@@ -465,6 +463,8 @@ def _eval_is_zero(self):
     def _eval_is_negative(self):
         return False
 
+
+
```

Comment:
PEP-8 states that there should only be a single blank line between class methods. Shouldn't we go with that methodology instead? That would mean removing all the weird triple line breaks in `floor`, `ceiling` and `frac`.

Reply:
It looks like PEP-8 conventions are not followed at other places in this file too, will fix those too.